modules/utils/claws/utils/box.hpp

```diff
@@ -22,7 +22,7 @@ namespace claws
     static_assert(!std::is_class_v<type>, "box should store a primitive, not a class");
 
   private:
-    type value{};
+    type _value{};
```

Comment:
Is there a specific reason for this renaming?

Reply:
I tend to mark my private things with something, usually `_` as a prefix or suffix.

That also gets rid of the member shadowing in the custom constructors (which is not bad per se, but could trigger warnings under some configurations)

Reply:
I don't like the noise of `_`, and I find it more logical to use `this` when shadowing occurs.
`